nuxeo-coldstorage-web/rollup.config.js

```diff
@@ -0,0 +1,67 @@
+import { terser } from 'rollup-plugin-terser';
+import copy from 'rollup-plugin-copy';
+import minifyHTML from 'rollup-plugin-minify-html-literals';
+import resolve from '@rollup/plugin-node-resolve';
+import path from 'path';
+
+// Replace imports by an export of existing globals in Web UI
+// https://github.com/nuxeo/nuxeo-web-ui/blob/master/index.js#L7
+const GLOBALS = {
+  '@polymer/polymer/lib/legacy/class.js': 'const { mixinBehaviors } = Polymer; export { mixinBehaviors };',
+  '@polymer/polymer/lib/legacy/polymer.dom.js':
+    'const { dom, html, matchesSelector } = Polymer; export { dom, html, matchesSelector };',
+  '@polymer/polymer/polymer-element.js': 'const { PolymerElement } = window; export { PolymerElement };',
+  '@polymer/polymer/lib/elements/dom-module.js': 'const { DomModule } = window; export { DomModule };',
+  '@polymer/polymer/lib/utils/html-tag.js': 'const { html } = Polymer; export { html };',
+  '@polymer/polymer/lib/utils/debounce.js':
+    'const { enqueueDebouncer, flushDebouncers, Debouncer } = Polymer; ' +
+    'export { enqueueDebouncer, flushDebouncers, Debouncer };',
+  '@nuxeo/nuxeo-ui-elements/nuxeo-i18n-behavior.js': 'const { I18nBehavior } = Nuxeo; export { I18nBehavior };',
+  '@nuxeo/nuxeo-ui-elements/nuxeo-filters-behavior.js':
+    'const { FiltersBehavior } = Nuxeo; export { FiltersBehavior };',
+  '@nuxeo/nuxeo-ui-elements/nuxeo-format-behavior.js': 'const { FormatBehavior } = Nuxeo; export { FormatBehavior };',
+};
+
+// Ignore these imports since they should just be all about custom element definitions which are done already by Web UI
+const IGNORES = [/^@(nuxeo|polymer)\//];
+
+const TARGET = 'target/classes/web/nuxeo.war/ui';
+
+export default {
+  input: 'index.js',
+  output: {
+    file: `${TARGET}/nuxeo-coldstorage.bundle.js`,
+    format: 'es',
+  },
+  plugins: [
+    copy({
+      targets: [{ src: 'i18n', dest: TARGET }],
+    }),
+    resolve(),
+    {
+      transform(code, id) {
```

Comment:
I read the [docs](https://github.com/rollup/rollup/blob/master/docs/05-plugin-development.md), but it was not clear to me where this is called. I assume this is called once per import and it modifies the line of the filewhere the import is happening. Is this true?